mkosi/__init__.py

```diff
@@ -6174,28 +6174,48 @@ def run_shell(args: CommandLineArguments) -> None:
         run(cmdline, stdout=sys.stdout, stderr=sys.stderr)
 
 
-def run_qemu(args: CommandLineArguments) -> None:
-    # Look for the right qemu command line to use
-    has_kvm = os.path.exists("/dev/kvm")
-    cmdlines: List[List[str]] = []
+def find_qemu_binary() -> str:
     ARCH_BINARIES = {"x86_64": "qemu-system-x86_64", "i386": "qemu-system-i386"}
     arch_binary = ARCH_BINARIES.get(platform.machine(), None)
```

Comment:
Not your change, but the `None` here is redundant, since `dict.get` will always return `None` on a missing key.